README.md

```diff
@@ -33,3 +33,83 @@ Your pull requests are welcome but please keep them manageable and focussed.
 Equally your input on any pull requests we have in flight at any given time is invaluable to us, so please get involved.
 
 Thanks! :grin:
+
```

Comment:
More than one line of vertical whitespace is never needed so please remove this additional line and the one below.

Also, can you insert this entire addition above 'Contributing' as is more conventional. As in we're helping users get up and running with the codebase before we encourage them to contribute!

Reply:
Done@https://github.com/ably/ably-flutter/commit/083a0a518b96c8e773f98c8353af420b698beaa3